contracts/ToucanRegenBridge.sol

```diff
@@ -112,6 +112,9 @@ contract ToucanRegenBridge is Ownable, Pausable {
         require(toucanContractRegistry.checkERC20(tco2), "not a TCO2");
         require(nctPool.checkEligible(tco2), "TCO2 not eligible for NCT pool");
 
+        uint256 precisionTest = (amount / 10**12) * 10**12;
```

Comment:
We can avoid making the exponentiations in the runtime by leveraging a `constant` `1e12`